src/utils/common.js

```diff
@@ -0,0 +1,28 @@
+/*  
+    classNames example usage:
+    <div className={classNames(['base-class', {'second' : someFlag, 'third' : true}])}></div>
+*/
+const classNames = (names) => {
+    let isArray = Array.isArray;
+    let i;
+    let len;
+    let tmp = typeof names;
```

Comment:
Let's use more descriptive names for variables, e.g. here it can be `let namesType = typeof names;`